EMIP/1-1000/EMIP00164.xml

```diff
@@ -38,11 +38,72 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                </msIdentifier>
                <msContents>
                   <summary>Homiliary in Honor of the Archangel Michael, ድርሳነ፡ ሚካኤል፡</summary>
-                  <msItem xml:id="ms_i1">
-                     <locus from="1r" to="88v"/>
-                     <title type="complete" ref="LIT1295Dersan"/>
-                     <textLang mainLang="gez">Gə‘əz</textLang>
-                  </msItem>
+                  <textLang mainLang="gez">Gə‘əz</textLang>
+               <msItem xml:id="ms_i1"><locus from = "1r" to = "88v"/><title type = "complete" ref = "LIT1295Dersan"/>
+               <note>
```

Comment:
I am not sure why this is a list in a note instead of a list of msItem elements.
https://betamasaheft.eu/Guidelines/?q=msItem&id=manuscriptContents